cmd/client/main.go

```diff
@@ -92,7 +94,18 @@ func loadConfig() (*config, *client.Config) {
 	}
 
 	clientCfg := client.GetDefaultConfig()
+
+	hostname, err := os.Hostname()
```

Comment:
Since the `os.Hostname()` value is just the default, please read (and set) it inside `GetDefaultConfig`.